.github/workflows/dist.yml

```diff
@@ -0,0 +1,48 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+name: 'Build distribution'
+
+on:
+  # Only run when release is created in the master branch
+  release:
+    types: [created]
+    branches:
+      - 'master'
+
+jobs:
+  build_sdist:
+    name: 'Build source distribution'
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v3
+        name: 'Checkout source repository'
+        with:
+            # You need to include this or setuptools_scm in GitHub runners won't detect the version
+            fetch-depth: 0
+
+      - uses: actions/setup-python@v4
+
+      - name: 'Build sdist'
+        run: python3 setup.py sdist
```

Comment:
This is highly discouraged by the PyPA. Setuptools is not an arbitrary command runner. Use pypa/build instead. Also, don't exclude the wheel.